Modules/System/Time Zone Management/app.json

```diff
@@ -0,0 +1,33 @@
+{
+  "id": "4c315253-b024-4fdc-9c3d-794036768af5",
```

Comment:
We've tried to stop calling things "management", as it really doesn't add much information to module. "Time Zone" would be fine, I guess?

Reply:
Perhaps, but given your previous comment where you would like to add codeunit 10 BaseApp functionality to it, would it be better to resolve this once agreement has been reached there?

Reply:
Absolutely.

Reply:
So, what should we agree upon? Should we make a "Date & Time" module, which contains a Timezone facade and implementation codeunit, containing both DateTimeOffset and DaylightSavingTimeInfo? It would be a more versatile module.
Input to that idea?

Reply:
I like the Date & Time module idea. 

To summarize and make sure we're on the same page: it will have a single timezone facade and implementation codeunit that merges the functionality that is now contained in the two separate facade/implementation pairs.

Reply:
Correct. We're exactly on the same page! I like that idea too. It has even grown on me since yesterday :-) Let's do that!

Reply:
Renamed the module and merged the face and implementation as discussed.